charts/seed-controlplane/charts/kube-apiserver/templates/_helpers.tpl

```diff
@@ -51,3 +51,22 @@
 {{- end }}
 {{- end }}
 {{- end -}}
+
+{{- define "kube-apiserver.admissionPluginConfigFileDir" -}}
```

Comment:
Why don't you put that in variable in `values.yaml` and not use templates?

Reply:
Because I didn't want that to be configurable (is there a reason for that)? Actually, most of the paths which we use in the charts are not put into those kind of "variables" but copied around where needed. We should anyway clean up the charts at some point in time.